sdk/scheduler/src/main/java/com/mesosphere/sdk/config/ConfigurationUpdater.java

```diff
@@ -16,20 +16,51 @@
      * The result of an {@link ConfigurationUpdater#updateConfiguration(Configuration)} call.
      */
     class UpdateResult {
+        public static final String LAST_COMPLETED_UPDATE_TYPE_KEY = "last-completed-update-type";
```

Comment:
May as well put this as a private const in StateStoreUtils as that's now the only thing using it